app/models/user.rb

```diff
@@ -35,6 +35,12 @@ class User < ApplicationRecord
     dependent: :destroy,
     inverse_of: 'invitee'
 
+  scope :invitee_select_for, ->(user) { User.excluding(user).pluck(:email, :id) }
+
+  def all_pair_requests
```

Comment:
i've been thinking what to rename this and maybe something like `my_pair_requests` is more explicit what it actually is?